backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/FilterData.kt

```diff
@@ -0,0 +1,24 @@
+package app.ehrenamtskarte.backend.stores.importer.nuernberg.steps
+
+import app.ehrenamtskarte.backend.stores.importer.ImportConfig
+import app.ehrenamtskarte.backend.stores.importer.PipelineStep
+import app.ehrenamtskarte.backend.stores.importer.nuernberg.types.CSVAcceptingStore
+import kotlinx.coroutines.runBlocking
+import org.slf4j.Logger
+
+class FilterData(config: ImportConfig, private val logger: Logger) :
+    PipelineStep<List<CSVAcceptingStore>, List<CSVAcceptingStore>>(config) {
+    override fun execute(input: List<CSVAcceptingStore>): List<CSVAcceptingStore> = runBlocking {
+        input.filter {
+            if (it.categoryId?.isEmpty()!!) {
+                logger.info("'${it.name}' was filtered out because category is null")
+                return@filter false
+            }
+            if (it.postalCode?.isEmpty()!! || it.houseNumber?.isEmpty()!! || it.location?.isEmpty()!!) {
+                logger.info("'${it.name}' was filtered out because location info is insufficient")
+                return@filter false
+            }
+            true
```

Comment:
Are we sure we want to filter that out? Maybe we can add it as an online store.

Reply:
so far we filter stores out for bavaria. only stores with `long/lat` gonna be written in DB